packages/@aws-cdk/aws-eks/README.md

```diff
@@ -45,6 +45,20 @@ cluster.addResource('mypod', {
 });
 ```
 
+### Endpoint Access
+
+You can configure the [cluster endpoint access](https://docs.aws.amazon.com/eks/latest/userguide/cluster-endpoint.html) by using the `endpointAccess` property:
+
+```typescript
+const cluster = new eks.Cluster(this, 'hello-eks', {
+  version: eks.KubernetesVersion.V1_16,
+  endpointAccess: eks.EndpointAccess.private() // No access outside of your VPC.
```

Comment:
Why isn't this a simple enum?

Reply:
To support passing `cidr` in `EndpointAccess.public` and `EndpointAccess.privateAndPublic`.